scripts/quests/windurst/Reap_What_You_Sow.lua

```diff
@@ -33,21 +33,14 @@ quest.sections =
 
         [xi.zone.WINDURST_WATERS] =
         {
-            ['Mashuu-Ajuu'] =
-            {
-                onTrigger = function(player, npc)
-                    if player:getFameLevel(xi.quest.fame_area.WINDURST) >= 4 then
-                        return quest:progressEvent(483)
-                    else
-                        return quest:progressEvent(463, 0, xi.items.SOBBING_FUNGUS, xi.items.BAG_OF_HERB_SEEDS)
-                    end
-                end,
-            },
+            -- Unsure where event(483) should be...
+            ['Mashuu-Ajuu'] = quest:progressEvent(463, 0, xi.items.SOBBING_FUNGUS, xi.items.BAG_OF_HERB_SEEDS),
 
             onEventFinish =
             {
                 [463] = function(player, csid, option, npc)
-                    if npcUtil.giveItem(player, xi.items.BAG_OF_HERB_SEEDS) and option == 3 then
+                    if option == 3 then
```

Comment:
it's good practice to leave npcUtil functions in the if statements as they return false if their action isn't possible.
Therefore if it's not possible to give them the item (Ex. their inventory is full, it's an ex item they already have) it won't do any other behavior - like starting the quest or progressing it in any way.
When things like this happen it can be awkward for GMs to diagnose what and where things went wrong.